app/views/devise/sessions/new.html.erb

```diff
@@ -1,26 +1,28 @@
-<h2>Log in</h2>
+<div class="grid-container">
```

Comment:
I see `grid-container` being used in multiple places, this could be something you apply around the `yield` in the application.html.erb file so that grid-container is applied to all yielded content. Not required, just a suggestion!